trview/Viewer.cpp

```diff
@@ -883,10 +691,10 @@ namespace trview
         {
             _level->set_selected_room(static_cast<uint16_t>(room));
 
-            _room_navigator->set_selected_room(room);
-            _room_navigator->set_room_info(_level->room_info(room));
+            _ui->set_selected_room(room);
+            _ui->set_room_info(_level->room_info(room));
```

Comment:
These could be combined in to one call for selecting a room - it can then do the number, info and load the minimap in one go.